src/logserver/server.py

```diff
@@ -112,6 +115,26 @@ async def handle_kafka_inputs(self):
             logger.info(f"Received message via Kafka:\n    ⤷  {value}")
             self.data_queue.put(value)
 
+    async def async_follow(self, file: str = "/opt/file.txt"):
```

Comment:
So to understand it correctly, it will always start reading from `/opt/file.txt` or is there a possibility to handle it via the `config.yml`?

Reply:
`opt/file.txt` is the name of the mounted file in the LogServer's Docker container, and therefore does not change. The entry in the `docker/.env` file (`MOUNT_PATH`) however can be changed by users, and this file is then mounted under the same name into the Docker container.

Reply:
But what about users who deploy the app as systemd service? See #6.

Reply:
Just solved that in the latest commit

Reply:
Great!